app/_src/gateway/admin-api/index_latest.md

```diff
@@ -146,6 +146,21 @@ curl -i -X POST http://localhost:8001/services/test-service/routes \
      -F "paths[2]=/path/two"
 ```
 
+## HTTP status response codes
+
+The following status codes are returned in HTTP responses:
+
+| HTTP Code | HTTP Description | Notes | Request method |
+| --------- | ---------------- | ----- | ------------- |
+| 200 | OK | The request succeeded. The result of a `200` request depends on the request type: `GET`: The resource was fetched and sent in the message body. `PUT` or `POST`:  The resource that describes the result of the action is sent in the message body. `PATCH`: ? | `GET`, `POST`, `PATCH`, `PUT` |
```

Comment:
```suggestion
| 200 | OK | The request succeeded. The result of a `200` request depends on the request type: <br>- `GET`: The resource was fetched and sent in the message body. <br>- `PUT` or `POST`:  The resource that describes the result of the action is sent in the message body. <br>- `PATCH`: ? | `GET`, `POST`, `PATCH`, `PUT` |
```

Reply:
@lena-larionova Thanks! I couldn't remember how we formatted those